web-portal/frontend/app/page.tsx

```diff
@@ -1,5 +1,10 @@
-import { Title } from "@mantine/core";
+import { HeroSection } from "@frontend/components/home / 01-Hero";
```

Comment:
What is up with the spacing here? I'm not familiar with this format.

Reply:
good catch